harvester/dmat/src/main/docker/harvester-dmat.json

```diff
@@ -1,3 +1,17 @@
 {
-  "appName": "DMatV2Harvester"
+  "appName": "DMatV2Harvester",
+  "jdbc": {
+    "jdbc/dataio/rawrepo-boblebad": {
+      "url": "${RAWREPO_BOBLEBAD_DB_URL}"
+    },
+    "jdbc/dataio/rawrepo-cisterne": {
+      "url": "${RAWREPO_CISTERNE_DB_URL}"
+    },
+    "jdbc/dataio/rawrepo-exttest": {
+      "url": "${RAWREPO_FBSTEST_DB_URL}"
+    },
+    "jdbc/dataio/basis-rawrepo": {
+      "url": "${RAWREPO_BASISMIG_DB_URL}"
+    }
+  }
```

Comment:
Hmm det er på een eller anden måde lidt bagvendt at skulle gå på databasen for at få udlveret url'en til recordservicen. Eftersom vi alligevel skal konfigurere db urlen kan vi vel ligesågodt konfigurere recordservice urlen direkte,

Reply:
problemet er at vi godt vil have 2 høstere kørende, en der høster til staging og en der høster til prod, og de skal hhv. høste fra cisterne og basismig (for at matche poster fundet i match i hhv. cisterne og basismig)

(staging vil nok være disablet en del af tiden, men vi har brug for den ind imellem)